emuu-client/src/components/NavbarPostLogin/AlgoliaSearchNavbar/AlgoliaSearchNavbar.js

```diff
@@ -0,0 +1,49 @@
+import React, { useRef } from "react";
+import "../HeaderPostLogin.scss";
+
+import Container from "react-bootstrap/Container";
+import Form from "react-bootstrap/Form";
+import Navbar from "react-bootstrap/Navbar";
+
+import ProfileMenu from "../../ProfileMenu/ProfileMenu";
+
+import EmuuLogo from "../EmuuLogo.png";
+
+const AlgoliaSearchNavbar = ({ autocomplete, searchInput }) => {
+  const inputFocusRef = useRef(null);
```

Comment:
Add comments to code. Something we all need to do for maintainability. @R313mon @Moamin1 @Tanvir-Rahman123